docs/SetupDatabaseForAzure.md

```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
 Note:-
 
 * The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for MySQL instance.
+
+
+## Azure Database for PostgreSQL
+
+In this section, you will create an Azure Database for PostgreSQL.
+
+### Steps
+
+You can choose Flexible or Single server deployment for Azure Database for PostgreSQL, follow the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to learn more.
+* To create an Azure Database for PostgreSQL, 
+  * For Single Server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal).
+  * For creating a flexible server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/quickstart-create-server-portal).
+* (Optional) Configure advanced monitoring services with the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/concepts-monitoring), monitoring is enabled in Azure Database for PostgreSQL by default.
+* (Optional) To scale resources like vCores and storage after the deployment,
+  * Follow the [Azure official guide](https://docs.microsoft.com/en-gb/azure/postgresql/concepts-pricing-tiers#scale-resources) for learning more about scaling single server resources.
```

Comment:
To learn, then it should `Read`. To do steps, it should `Follow`.  Which is correct?

Reply:
Sorry for the mistake
`Read` will be correct considering the document linked here.
I have updated the sentence.